src/sagemaker_training/torch_distributed.py

```diff
@@ -0,0 +1,163 @@
+# Copyright 2018-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License'). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the 'license' file accompanying this file. This file is
+# distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""This module contains functionality related to Torch Distributed Elastic Runner.
+Refer: https://pytorch.org/docs/stable/elastic/run.html
+"""
+from __future__ import absolute_import
+
+import os
+import sys
+
+from sagemaker_training import _entry_point_type, environment, errors, logging_config, process
+
+TORCH_DISTRIBUTED_MODULE = "torchrun"
+MASTER_PORT = "7777"
+RDZV_BACKEND = "c10d"
```

Comment:
Why are we having this. We don't seem to be using anywhere in the torchrun cmd. Can we remove this if not used?

Reply:
Updated.